### ISSUE TITLE
Crash on click on attribution (!) mark

### ISSUE THREAD

**Issue by dryaz** [mapbox/mapbox-maps-android#2178]
## Environment

- Android OS version: Api lvl 32
- Devices affected: Emulator
- Maps SDK Version: 11.0.0-beta.3

## Observed behavior and steps to reproduce

Click on attribution (!) icon == Crash

## Expected behavior

No crash

## Notes / preliminary analysis

Current stable 10.16.0 works fine

## Additional links and references

FATAL EXCEPTION: main
                                                                                                    Process: io.manymap.app.android, PID: 5509
                                                                                                    android.content.res.Resources$NotFoundException: Resource ID #0x0
                                                                                                    	at android.content.res.ResourcesImpl.getValue(ResourcesImpl.java:225)
                                                                                                    	at android.content.res.Resources.loadXmlResourceParser(Resources.java:2446)
                                                                                                    	at android.content.res.Resources.getLayout(Resources.java:1252)
                                                                                                    	at android.view.LayoutInflater.inflate(LayoutInflater.java:536)
                                                                                                    	at android.view.LayoutInflater.inflate(LayoutInflater.java:485)
                                                                                                    	at androidx.appcompat.app.AlertController$AlertParams.createListView(AlertController.java:989)
                                                                                                    	at androidx.appcompat.app.AlertController$AlertParams.apply(AlertController.java:965)
                                                                                                    	at androidx.appcompat.app.AlertDialog$Builder.create(AlertDialog.java:984)
                                                                                                    	at androidx.appcompat.app.AlertDialog$Builder.show(AlertDialog.java:1008)
                                                                                                    	at com.mapbox.maps.plugin.attribution.AttributionDialogManagerImpl.showAttributionDialog(AttributionDialogManagerImpl.kt:87)
                                                                                                    	at com.mapbox.maps.plugin.attribution.AttributionDialogManagerImpl.showAttribution(AttributionDialogManagerImpl.kt:54)
                                                                                                    	at com.mapbox.maps.plugin.attribution.AttributionPluginImpl.onClick(AttributionPluginImpl.kt:56)
                                                                                                    	at android.view.View.performClick(View.java:7455)
                                                                                                    	at android.view.View.performClickInternal(View.java:7432)
                                                                                                    	at android.view.View.access$3700(View.java:835)
                                                                                                    	at android.view.View$PerformClick.run(View.java:28810)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:938)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:201)
                                                                                                    	at android.os.Looper.loop(Looper.java:288)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:7870)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)

**Comment by dryaz** [comment]
It's happening 'cause of 
```
<style name="AppTheme" parent="android:Theme.Material.NoActionBar"/>
```

`AttributionDialogManagerImpl` uses AppCompat dialog to show attribution dialog which requirs AppCompat theme.

**Comment by yunikkk** [comment]
@dryaz thanks for the report!

**Comment by kiryldz** [comment]
This should be fixed in upcoming v11.0.0-beta.6 coming out tomorrow. Please re-open if the issue persists.
